.github/CODEOWNERS

```
@@ -0,0 +1,2 @@
+# CI
+.github/**/*		@iHiD @ErikSchierboom
```

This is the team I've made based on our discussions earlier.

```suggestion
.github/**/*		@exercism/maintainers-admin
```